.github/workflows/push.yml

```diff
@@ -46,3 +46,6 @@ jobs:
       run: |
         docker login -u ${{ secrets.DOCKER_USER }} -p ${{ secrets.DOCKER_PASSWORD }}
         ./hack/docker-push.sh
+
+        # Push image to GCR to facilitate some environments that have rate limitation to docker hub, e.g. vSphere.
+        REGISTRY=gcr.io/velero-gcp ./hack/docker-push.sh
```

Comment:
How do we handle the GCR credential used to push?

Reply:
I missed this part. I did the verification in my already setup environment, so I didn't met the issue.
I will add GCP related action to do the job.